### PR TITLE
Increase code coverage for pinot-java-client

### PR DESCRIPTION
**Pull request by ayush-bajaj**
## Description
Issue link: https://github.com/apache/incubator-pinot/issues/5694
Increased the code coverage for the pinot-java-client module to more than 75%.

### CODE REVIEW

File: pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/DynamicBrokerSelector.java [hunk 1]
```diff
@@ -88,6 +96,7 @@ public void handleDataChange(String dataPath, Object data)
     refresh();
   }
 
+
```
Comment (jackjlli) [inline pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/DynamicBrokerSelector.java:99]: Extra line

File: pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/DynamicBrokerSelectorTest.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    // Run the test
+    final String result = dynamicBrokerSelectorUnderTest.selectBroker("table1");
+
+    // Verify the results
+    assertEquals("broker1", result);
+  }
+
+  @Test(priority = 1)
+  public void testSelectBroker_nullTable() throws Exception {
```
Comment (jackjlli) [inline pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/DynamicBrokerSelectorTest.java:98]: It's recommended to use camel case in the method name. So please avoid using underscore here.

File: pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/ExternalViewReaderTest.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+    // Run the test
+    final List<String> result = externalViewReaderUnderTest.getLiveBrokers();
+
+    // Verify the results
+    assertEquals(expectedResult, result);
+  }
+
+  @Test
+  public void testGetLiveBrokers_exception() throws IOException {
```
Comment (jackjlli) [inline pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/ExternalViewReaderTest.java:75]: Avoid using underscore in the method name

**Review by jackjlli — APPROVED** [review]
Thanks for taking this up! Minor but LGTM.

File: pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/DynamicBrokerSelectorTest.java (hunk 2 above)
Comment (ayush-bajaj) [reply pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/DynamicBrokerSelectorTest.java:98]: https://stackoverflow.com/questions/155436/unit-test-naming-best-practices

Hey this is the approach I follow for keeping unit test names, I found out during my work that it's acceptable to use an underscore to seperate the following: UnitOfWork_StateUnderTest, but only for unit tests and not for actual function names. 
I can still make the change tho to keep it in accordance with the project. should I go ahead with the changes?

Comment (ayush-bajaj) [reply pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/DynamicBrokerSelectorTest.java:98]: https://stackoverflow.com/questions/155436/unit-test-naming-best-practices

Hey this is the approach I follow for keeping unit test names, I found out during my work that it's acceptable to use an underscore to seperate the following: UnitOfWork_StateUnderTest, but only for unit tests and not for actual function names.
I can still make the change tho to keep it in accordance with the project. should I go ahead with the changes?

Comment (jackjlli) [reply pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/DynamicBrokerSelectorTest.java:98]: It'd be good to keep the naming format consistent for all the tests within one single repo. I'd suggest so to make less confusion.

**Review by jackjlli — APPROVED** [review]
LGTM. Thanks for making the code more robust!

**Pull request merged**